common/src/main/java/com/scottlogic/deg/common/commands/CommandBus.java

```diff
@@ -0,0 +1,25 @@
+package com.scottlogic.deg.common.commands;
+
+import javafx.util.Pair;
```

Comment:
I'd advise avoiding `javafx.util.Pair`, as this module is broken out of the main JDK in later versions of the JDK. We could either create our own, or use Apache Commons lang3 library's version.

Reply:
I have now replaced this